notebooks/Repo_Analysis.ipynb

```diff
@@ -33,6 +33,8 @@
    ],
```

Comment:
<p>Do we want to do this every time? or only an an initial run?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-services-group/metrics/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Yep, we wanna create tables just on the initial run. This func will get called every time but I believe it won't create tables again coz it uses `CREATE TABLE IF NOT EXISTS`